django_forest/utils/to_bool.py

```diff
@@ -0,0 +1,12 @@
+TRUE_VALUES = {
+    't', 'T',
+    'y', 'Y', 'yes', 'YES',
+    'true', 'True', 'TRUE',
+    'on', 'On', 'ON',
+    '1', 1,
+    True
+}
+
+
+def to_bool(value):
```

Comment:
You can use [distutils.util.strtobool](https://docs.python.org/3/distutils/apiref.html#distutils.util.strtobool).
Just put a try/catch as it throws with "invalid" values.

Reply:
Tested with python 2.7 and 3.8.

Reply:
right! that's great!
thanks!